src/constants.F90

```diff
@@ -127,6 +127,9 @@ module constants
        SURF_CONE_Y = 10, & ! Cone parallel to y-axis
        SURF_CONE_Z = 11    ! Cone parallel to z-axis
 
+  ! Flag to say that the outside of a lattice is not defined
+  integer, parameter :: NO_OUTER_UNIV = -22
```

Comment:
I'd prefer the slightly more verbose `NO_OUTER_UNIVERSE`. And is there a reason you chose -22 as opposed to, say, -1?